src/interpreter.rs

```diff
@@ -62,8 +66,8 @@ mod tests {
 
     #[test]
     fn register_word() {
-        let mut interpreter = Interpreter::<i32>::default();
         let plus = fixture_plus();
+        let mut interpreter = Interpreter::<i32>::default();
         interpreter.define_word("+", &plus);
```

Comment:
That one is tricky: since the interpreter will contain a reference to `plus`, it cannot have a longer lifetime than `plus`